src/nnet3bin/nnet3-egs-augment-image.cc

```diff
@@ -289,11 +314,13 @@ int main(int argc, char *argv[]) {
 
 
     int32 srand_seed = 0;
+    std::string fill_mode_string = "nearest";
 
     ImageAugmentationConfig config;
 
     ParseOptions po(usage);
     po.Register("srand", &srand_seed, "Seed for the random number generator");
+    po.Register("fill_mode", &fill_mode_string, "Mode for filling the out-bundaries points = {nearest, reflect}");
```

Comment:
use "fill-mode" 
typo: bundaries should be boundaries;
and try to  stay within the google-style-guide-mandated 80-character limit.  e.g.
```
po.Register("fill-mode", &fill_mode_string, "Mode for dealing with "
                  "points outside the image boundary when applying transformations. "
                  "Choices = {nearest, reflect}");
```